llvm/test/MC/AMDGPU/vinterp.s

```diff
@@ -0,0 +1,236 @@
+// RUN: llvm-mc -triple=amdgcn -mcpu=gfx1100 -mattr=+real-true16 -show-encoding %s | FileCheck -check-prefix=GCN %s
```

Comment:
Why doesn't this use your script updating MC tests?

It would help if you could pre-commit this by copying vinterp-fake16.s as you did previously with other true16/fake16 tests.

Reply:
Updated with the update_mc script